test/benchmarks/standalone/benchmark_text_input.cc

```diff
@@ -91,7 +91,7 @@ static void benchmark_cb_adf_large(
   std::string cmd = "--cb_explore_adf --quiet";
   if (interactions) { cmd += " -q ::"; }
   auto vw = VW::initialize(cmd, nullptr, false, nullptr, nullptr);
-  int example_size = 100;
+  int example_size = 10;
```

Comment:
Whats the reason behind this change?

Reply:
I've been seeing the benchmarking CI fail a lot recently, and thought it may have to do with these new benchmarks I added being too large

Reply:
Ah yeah those are tricky. Cool lets see if this helps